// api/mockItems.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "Букет Чувашии",
    description: "Пиво имеет тончайший букет ароматов и чистый благородный цвет. Это пиво не просто пьют, а наслаждаются, получают удовольствие от каждого глотка. Удачное сочетание компонентов делают его вкус неповторимым. Вкус отмечается приятным хмелевым ароматом в сочетании с выраженной хмелевой горечью и винным привкусом. Элитное пиво для солидных людей - ценителей изысканного вкуса. Представительское пиво.",
    price: 48,
    image: "/uploads/image1.jpg",
    alco: "6.8 %",
    caloric: "62 ккал"
  },
  {
    id: "2",
    categoryId: "1",
    name: "Чебоксарское",
    description: "Светлое пиво, приготовленное по старинным рецептам, с соблюдением чувашских традиций. Полный насыщенный специфический вкус и аромат сброженного солодового напитка с мягкой хмелевой горечью и хмелевым ароматом. Сочетание разных сортов солода делает вкус неповторимым.",
    price: 42,
    image: "/uploads/image2.jpg",
    alco: "5.1 %",
    caloric: "50 ккал"
  },
  {
    id: "3",
    categoryId: "1",
    name: "Праздничный Букет",
    description: "Пиво сварено согласно немецкому закону о «чистоте пивоварения». Сочетание в нем мягкой хмелевой горечи и солодового вкуса делают этот напиток оригинальным, а мелкая пышная пена и янтарный вид  - запоминающимся. Пиво с такой пеной обладает полнотой вкуса и долго сохраняет свою свежесть.",
    price: 58,
    image: "/uploads/image3.jpg",
    alco: "5 %",
    caloric: "50 ккал"
  },
  {
    id: "4",
    categoryId: "1",
    name: "Визитное",
    description: "Светлое пиво, обладает мягким сброженным солодовым вкусом и свежим ароматом светлого солода и хмеля.",
    price: 42,
    image: "/uploads/image4.jpg",
    alco: "5.1 %",
    caloric: "50 ккал"
  },
  {
    id: "5",
    categoryId: "1",
    name: "Живое",
    description: "Светлое непастеризованное пиво. Пиво с приятным хмелевым ароматом и хмелевой горечью. Очень полезно для здоровья, так как содержит частицы живых микроорганизмов и дрожжей. Это пиво придаст вам сил и уверенности в себе.",
    price: 40,
    image: "/uploads/image5.jpg",
    alco: "4.7 %",
    caloric: "45 ккал"
  },
  {
    id: "6",
    categoryId: "2",
    name: "Пенное Золото",
    description: "Светлое пиво с освежающим вкусом и ароматом сброженного солодового напитка с мягкой хмелевой горечью и хмелевым ароматом. Имеет красивый золотистый цвет с плотной  стойкой пеной.",
    price: 49,
    image: "/uploads/image6.jpg",
    alco: "5.2 %",
    caloric: "50 ккал"
  },
  {
    id: "7",
    categoryId: "2",
    name: "Пенное Серебро",
    description: "Светлое пиво, приготовлено по немецкой технологии. Вкус и аромат сброженного солодового напитка с мягкой хмелевой горечью. Ровная, плотная пена придает неповторимый вид и покоряет любителей пива.",
    price: 46,
    image: "/uploads/image7.jpg",
    alco: "4.7 %",
    caloric: "46 ккал"
  },
  {
    id: "8",
    categoryId: "3",
    name: "Пражское",
    description: "Пиво «Пражское пастеризованное» — светлое пастеризованное фильтрованное пиво. Пиво легкое, имеет горьковато-хмелевой вкус. Рис, имеющийся в рецептуре, придает пиву мягкий вкус.",
    price: 44,
    image: "/uploads/image8.jpg",
    alco: "4.8 %",
    caloric: "46 ккал"
  },
  {
    id: "9",
    categoryId: "3",
    name: "Волжские зори",
    description: "Это пиво светло-янтарного цвета.  Ячмень добавляют  для получения более полного тела пива, усиления зернового вкуса и повышения пеностойкости.",
    price: 42,
    image: "/uploads/image9.jpg",
    alco: "4.5 %",
    caloric: "42 ккал"
  },
  {
    id: "10",
    categoryId: "3",
    name: "Жигулевское",
    description: "Это пиво-история! «Жигулевское» было самой популярной маркой пива в СССР. Про него пел Высоцкий: «Не дам порочить наш советский городок, где пиво варят золотое «Жигули»!!!» Про него снимали фильмы и писали книги. Это традиционное пиво, проверенное временем, с приятным солодовым и хмелевым вкусом времен «застоя». Особо хорошо пьется с воблой или варенными раками…",
    price: 46,
    image: "/uploads/image10.jpg",
    alco: "4.5 %",
    caloric: "42 ккал"
  }
]
